kernel/build/config-amd64

```diff
@@ -1631,7 +1637,6 @@ CONFIG_IP_VS_NFCT=y
 # IP: Netfilter Configuration
 #
 CONFIG_NF_DEFRAG_IPV4=y
-CONFIG_IP_NF_IPTABLES_LEGACY=y
```

Comment:
Dropping legacy iptables, hopefully nothing is using it due to conflict with nftables which is in use by Talos